tests/test_iterations.py

```diff
@@ -207,6 +207,9 @@
 )
 @pytest.mark.parametrize("n_iterations", [3], ids=["i3"])
 def test_iter_mc_acquisition_function(campaign, n_iterations, batch_size, acqf):
+    if isinstance(acqf, qKG):
+        pytest.xfail("qKG only works with continuous spaces.")
```

Comment:
* I guess strictly speaking this should rather be a "skip" than an "xfail", no? Skip = shouldn't even be tested, xfail = should actually succeed but currently fails due to other unresolved problems
* Let's not hardcode "qKG" here, even though we are just in tests

Reply:
aye about the xfail
but why the second point?

Reply:
@AdrianSosic implementing this would require to remove the option `--error-for-skips` are you fine with that?

Reply:
Hm, I see, hadn't thought of that 🤔 ... What do you think? Probably the only way if we don't want to have `x`s show up all over the place in our test overview 😢 

In case you want to go for it, two comments:
* don't forget to remove the optional dep then
* there is also a `pytest.skipif` method that allows you to directly include the condition

Reply:
`pytest.skipif` does not exist, its only a marker, but I need an action here, so I can only got with `skip`
Ive also tried marking the corresponding acqfs, but this causes downstream problems because once wrapped in a marker you cannot use our `.is_mc` or `.abbreviation` calls anymore, so for now its a simple skip

skip dependency now removed